### PR TITLE
Combed through anti-derivs. Adjusted Figure widths in def. ints

### PR DESCRIPTION
**Pull request by CVollet**

### CODE REVIEW

**Comment by CVollet** [comment]
I think the whitespace issue Alex fixed before is causing a conflict. Also, we need to decide if we want some sort of style guidelines for definite integrals. I've left a lot of the images as is for right now.

**Comment by CVollet** [comment]
I'll be moving onto Chapter 6 after this (I'll come back to the rest of Chapter 5). I want to try to stay ahead of where I am in class right now.

File: src/sec_def_int.mbx [hunk 1]
```diff
@@ -285,7 +283,7 @@
                   <description></description>
                   <latex-image-code>
                     <![CDATA[
-                    \begin{tikzpicture}
```
Comment (Alex-Jordan) [inline src/sec_def_int.mbx]: Looks like a line was deleted here that should stay.

Comment (CVollet) [reply src/sec_def_int.mbx]: Probably when I was cleaning up the merge conflict. Fixed now.

**Comment by CVollet** [comment]
I think @Alex-Jordan had looked at this one before and only found one error (since fixed). Can we merge this one?

**Pull request merged**